install/migrations/update_10.0.x_to_11.0.0/assets.php

```diff
@@ -220,4 +220,8 @@
 foreach ($assignable_asset_rights as $rightname) {
     $migration->addRight($rightname, READ_ASSIGNED, [$rightname => READ]);
     $migration->addRight($rightname, UPDATE_ASSIGNED, [$rightname => UPDATE]);
+    if ($rightname !== 'cartdrige' && $rightname !== 'consumable') {
```

Comment:
@orthagh To simplify the code, could we add `users_id` and `groups_id` fields to the cartridges/consumables items ?

Reply:
Based on my previous comment maybe a Group field makes sense, but I don't think it makes sense to assign an entire cartridge classification to a user.

Reply:
make all itemtypes generic, all should have the 4 fields.